### ISSUE TITLE
Map für roidmi, viomi und dreame

### ISSUE THREAD

**Issue by MeisterTR** [iobroker-community-adapters/ioBroker.mihome-vacuum#725]
Hi, ich habe gesehen dass bei HA auch die Maps für die anderen Hersteller (nicht roborock) unterstützt werden vll eine coole Sache für die 4.0
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#supported-devices

https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/blob/master/custom_components/xiaomi_cloud_map_extractor/dreame/map_data_parser.py

https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/blob/master/scripts/map_processor.py

**Comment by DemigodCode** [comment]
Ja, bin da gerade dran. Das ist aber nicht sooo trivial. Vor allem weil man die Map scheinbar nur noch aus der Cloud kriegt. Direkt vom Robot nicht.

Ich baue da gerade einen allgemeingültigen Cloud Connector für Xiaomi Cloud V2.

**Comment by xXBJXx** [comment]
Closing this as a duplicate of #823, which tracks the same topic (map and room support for Dreame models) and is where the current status and any progress will be posted. Please continue there and subscribe to it for updates; nothing from this thread is lost, the relevant details are referenced in #823.

For Viomi there is no map data source the adapter could read, so a Viomi map is not planned.
